### ISSUE TITLE
Release 9.1.1-1 produces a warning when applied to RC 9.2.2

### ISSUE THREAD

**Issue by pbchase** [ctsit/shibboleth_and_table_auth#6]
Release 9.1.1-1 produces a warning when applied to RC 9.2.2. This is the full text of a successful patch:
```
→ bash apply_patch.sh redcap9.2.2.zip 
Unzipping redcap9.2.2.zip into a temporary folder
Please be patient, this may take a few minutes
patching file Classes/Authentication.php
patching file Config/init_functions.php
Hunk #1 succeeded at 457 with fuzz 2.
patching file ControlCenter/create_user.php
patching file ControlCenter/header.php
patching file ControlCenter/security_settings.php
REDCap redcap9.2.2.zip was successfully patched
Zipping patched redcap into output/redcap9.2.2.zip
Please use the patched zip file, output/redcap9.2.2.zip, as you normally would.
```
Could you update the patch for RC 9.2.2?

**Comment by ChemiKyle** [comment]
https://github.com/ctsit/shibboleth_and_table_auth/blob/c3bd08a2db58e0bb72373e23648706beee2d5d10/apply_patch.sh#L10

We might want to change this to an additional, optional arg. Or, to be really nice, `REDCAP_PATCH_VERSION` could be determined by using `REDCAP_VERSION` as a base and looking in the root dir for the nearest `redcap-patch*.patch` <= `REDCAP_VERSION`

**Comment by ChemiKyle** [comment]
Pending review of [this PR](https://github.com/ctsit/shibboleth_and_table_auth_dev/pull/9), a 9.2.2 patch is ready.

**Comment by pbchase** [comment]
Addressed in release 9.2.2-1
